Further Kirby Adjustments

ethangarsia517 opened the pull request:
# Changelog
## Kirby
### General
```diff
+ [+] Probability of losing Copy Ability: 0.667 -> 0.6
# Probability is ((value * damage taken in a single hit)/100).
```

### Dash Attack
```diff
Late Hit:
- [-] Angle: 65 -> 62
- [-] BKB: 45 -> 50
# Decreased reward for combo potency 
```

### **Back Aerial**
```diff
Early Hit:
- [-] Damage: 14% -> 13%
~ [/] BKB: 15 -> 19
- [-] KBG: 108 -> 104
# Power decreased for 50/50 purposes with Up Aerial, still decent finisher
```

### Up Aerial
```diff
Late Hit:
~ [$] Adjusted sfx
```

### Neutral Special (Inhale)
```diff
- [-] Minimum Frames opponent is held: 17 -> 12
- [-] Maximum Frames before grab release: 600 -> 300
```

### Neutral Special (Pikmin Toss)
```diff
+ [+] Reverted Max Pikmin at a time: 2 -> 3
```

### Hammer Flip
```diff
! [R] All hitboxes when not fully charged can no longer reverse hit
! [!] Changed handle to weak hit with increased priority when not fully charged

Ground:
- [-] Startup increased by 1F
- [-] FAF (Not Fully Charged): 55 -> 60
- [-] FAF (Fully Charged) 60 -> 65

Ground (Weak Hit):
~ Damage: 19 -> 12%
~ BKB: 60 -> 68
~ KBG: 78 -> 62
~ Angle: 48 -> 63
~ Hitstun: 1.1 -> 1.0

Ground (Fully Charged)
- [-] Size (Handle): 3.9 -> 3.6
- [-] Damage: 35% -> 32%
- [-] BKB: 60 -> 55
- [-] KBG: 78 -> 72

Air (Strong Hit 1)
+ [+] Damage: 11% -> 12%
~ [/] BKB: 67 -> 60
- [-] KBG: 106 -> 102

Air (Weak Hit 1)
~ Damage: 12% -> 10%
~ BKB: 67 -> 67
~ KBG: 106 -> 78
~ Angle: 50 -> 65

Air (Strong Hit 2) -
- [-] KBG: 94 -> 92

Air (Weak Hit 2) -
~ Damage: 10% -> 8.5%
~ BKB: 67 -> 67
~ KBG: 72
~ Angle: 38 -> 53

# Albeit having decreased priority, strong hits are still easy to hit but may require better spacing. With enough and damage charge even the weak hits can KO.
```

### Stone
```diff
- [-] Changed to Once Per Airtime, including when initially used grounded.
! [!] Gravity and air drift is enabled on Frame 14 when transforming out of Stone.
- [-] Heavy Armor (Stone): 25% -> 15%
+ [+] Knockback Multiplier when Stone Heavy Armor is broken: 1.0 -> 0.5

Ground:
+ [+] Startup: 10F -> 9F
- [-] Maximum Frames in Stone on the ground: 150F -> 90F
+ [+] Shield Damage when started on the ground: 13 -> 16
+ [+] Shield Damage when started in the air: 6 -> 13
 

Air:
~ [/] Fall Speed: 4.5 -> 3.9
- [-] Shield Damage when started in the air: 3 -> 0
- [-] Shield Damage when pushed off an edge: 3 -> 0

# Getting both hits on shield when starting in the air went from 41 -> 45 total. Using it on the ground went from 27 -> 30.
# Stone time is infinite in the air and does not count down until hitting the ground.
# Enabling gravity sooner means Kirby cannot perform an aerial out of Stone if he was grounded. But, it sort of land cancels and transitions nicely into grounded options if not jumped or airdodged out of.
# For now, these changes reduce stalling, encourage challenging Stone on the ground and makes the interaction faster paced, while making it riskier if Kirby is close to an unpunished enemy at the end of the move.
```

SkewedAskew committed:
Corn

SkewedAskew committed:
Koopa

SkewedAskew committed:
Wario

TeamPearson committed:
sus

SuddyN committed:
Merge branch 'pre-release' into corrin

SuddyN committed:
corrin DSpecial momentum rewrite

SuddyN committed:
Merge branch 'pre-release' into koopa

SuddyN committed:
dtilt1 small buffs

SuddyN committed:
FSmash 50% reversion

SuddyN committed:
UAir/FAir damage reversion

SuddyN committed:
Merge branch 'pre-release' into wario

SuddyN committed:
FAir hitbox partial reversion

SuddyN committed:
waft late hit nerf

SuddyN committed:
match DAir momentum handling to Pac Man

SuddyN committed:
Update aerials.rs

SuddyN committed:
alternative corrin fsmash

SuddyN committed:
Merge pull request #318 from HDR-Development/wario

Wario adjustments

SuddyN committed:
Merge pull request #313 from HDR-Development/koopa

Bowser adjustments

SuddyN committed:
Merge pull request #359 from HDR-Development/Falco-dsmash

sus

SuddyN committed:
Merge branch 'dev' into pre-release

SuddyN committed:
Merge branch 'pre-release' of https://github.com/HDR-Development/HewDraw-Remix into pre-release

SuddyN committed:
Merge branch 'pre-release' into v0.49.3-prerelease-branch

SuddyN committed:
Merge pull request #372 from HDR-Development/v0.49.3-prerelease-branch

v0.49.3 prerelease branch

SuddyN committed:
Merge branch 'dev' into pre-release

SuddyN committed:
Merge branch 'dev' into pre-release

SuddyN committed:
fix code duplication

SuddyN committed:
Revert "alternative corrin fsmash"

This reverts commit 616c4a8b9a729ad8978ba3d7f134f78f6953c45f.

SuddyN committed:
Merge pull request #312 from HDR-Development/corrin

Corrin adjustments

ethangarsia517 committed:
the time for poyo has come

SuddyN commented:
add "diff" to the start of the code block to get buff/nerf highlighting

example:
\`\`\`diff

\`\`\`

ethangarsia517 committed:
corrections

not sure why I made dash attack angle 68 at first thats a wholeass buff LOL, correct speciallw shield damage and decreased heavy armor percent

WuBoytH committed:
Adjustments. Again. (#379)

* Fix gut check momentum when sliding off of stage

* disable kirby upB startup ledgegrab

* extend zss forward ledgegrab range

* fix chrom USpecial nuking ppl

* fix SSpecial4S effects

* Fix Geyser Teleports

* fix bayo copy ability?

---------

Co-authored-by: jobrien97 <47401664+jobrien97@users.noreply.github.com>
Co-authored-by: SuddyN <suddy121800@hotmail.com>
Co-authored-by: jaccob101 <jacobking101007@gmail.com>

SuddyN committed:
Merge branch 'dev' into pre-release

ethangarsia517 committed:
Merge branch 'pre-release' into morepoyoadjustment

ethangarsia517 committed:
Merge branch 'pre-release' of https://github.com/HDR-Development/HewDraw-Remix into pre-release

ethangarsia517 committed:
Merge branch 'pre-release' into morepoyoadjustment
